DeAceroDLabs/marketplace-frontend
# fix: [DMP-533] Added formula so that javascript can handle decimals.

## What is done

[[DMP-533] Added formula so that javascript can handle decimals.](https://deacero.atlassian.net/browse/DMP-533)

## How is done

- Added formula so that typescript can handle decimals. This formula will be perform after each sum to avoid having a big decimal error from typescript.
- Round up last 2 decimals of each product price (This is optional).
- The app file was changed just for testing.

## How to test

1. Run the project via `npm start`.
2. Clear cache.
3. /carrito/productos.
4. You should see the correct sum of products.
